src/App.css

```diff
@@ -1,38 +1,5 @@
 .App {
```

Comment:
Probably you don't need this file at all. It's default CRA style file and you should use your BEM like or css module like styles.